cosmwasm/contracts/crosschain-registry/src/state.rs

```diff
@@ -28,6 +29,7 @@ impl StorageKey {
             StorageKey::GlobalAdminMap => "gam",
             StorageKey::ChainAdminMap => "cam",
             StorageKey::ChainMaintainerMap => "cmm",
+            StorageKey::HasPacketForwardMiddleware => "hpfm",
```

Comment:
General question, is this used as the key in cosmwasm, or is this just for debugging? If its for debugging can we use bigger names?

Reply:
it's used as a key